### PR TITLE
Edit Currency Custom Website

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
**Description**
This PR with a test that covers Custom Website Currency editing

**Preconditions:**
 1. Setup configuration of main website.
 2. Create custom website.
 3. Create product.
 4. Assign created product to the main and custom websites.

**Steps To reproduce:**
1. Setup configuration of custom website.
2. Open created product on main website
3. Open created product on custom website
4. Perform asserts.

Fixed issues: #583  Convert EditCurrencyCustomWebsiteTest to MFTF

### CODE REVIEW

File: app/code/Magento/CurrencySymbol/Test/Mftf/Test/AdminEditCurrencyCustomWebsiteTest.xml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            <magentoCLI command="config:set {{StorefrontDisableAddStoreCodeToUrls.path}} {{StorefrontDisableAddStoreCodeToUrls.value}}" stepKey="setAddStoreCodeToUrlsToNo"/>
+            <deleteData createDataKey="createSimpleProduct" stepKey="deleteSimpleProduct"/>
+            <actionGroup ref="DeleteCustomWebsiteActionGroup" stepKey="deleteCustomWeWebsite">
+                <argument name="websiteName" value="customWebsite.name"/>
+            </actionGroup>
+            <magentoCLI stepKey="cleanCache" command="cache:clean"/>
+            <actionGroup ref="logout" stepKey="logout"/>
+        </after>
+
+
```
Comment (eduard13) [inline app/code/Magento/CurrencySymbol/Test/Mftf/Test/AdminEditCurrencyCustomWebsiteTest.xml]: Remove the not needed empty line

File: app/code/Magento/CurrencySymbol/Test/Mftf/Test/AdminEditCurrencyCustomWebsiteTest.xml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        <!-- TEST BODY -->
```
Comment (eduard13) [inline app/code/Magento/CurrencySymbol/Test/Mftf/Test/AdminEditCurrencyCustomWebsiteTest.xml]: Also don't think that it's really needed to specify the TEST BODY here. May you also remove it please?

**Comment by DmitryTsymbal** [comment]
Hey @eduard13 . Thank you for the review!
I have made requested changes. Could you please have a look?
Thanks!

File: app/code/Magento/CurrencySymbol/Test/Mftf/Test/AdminEditCurrencyCustomWebsiteTest.xml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminEditCurrencyCustomWebsiteTest">
+        <annotations>
+            <features value="CurrencySymbol"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/CurrencySymbol/Test/Mftf/Test/AdminEditCurrencyCustomWebsiteTest.xml]: You can use Human - friendly form there :-)

File: app/code/Magento/CurrencySymbol/Test/Mftf/Test/AdminEditCurrencyCustomWebsiteTest.xml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                <argument name="customStore" value="customStore"/>
+            </actionGroup>
+            <createData entity="ApiSimplePrice100Qty100v2" stepKey="createSimpleProduct"/>
+            <actionGroup ref="FilterAndSelectProductActionGroup" stepKey="openAdminProductPage">
+                <argument name="productSku" value="$$createSimpleProduct.sku$$"/>
+            </actionGroup>
+            <actionGroup ref="AdminAssignProductInWebsiteActionGroup" stepKey="assignProductToSecondWebsite">
+                <argument name="website" value="{{customWebsite.name}}"/>
+            </actionGroup>
+            <actionGroup ref="SaveProductFormActionGroup" stepKey="clickSaveButton"/>
```
Comment (lbajsarowicz) [inline app/code/Magento/CurrencySymbol/Test/Mftf/Test/AdminEditCurrencyCustomWebsiteTest.xml]: Due to the fact that some Tests environments are running on indices set `On schedule` update, please add `<magentoCron stepKey="runCronIndex" groups="index"/>` to trigger indices partial reindex.

**Pull request merged**